src/CoreBundle/private/modules/MTHeader/MTHeader.class.php

```diff
@@ -64,12 +64,6 @@ public function &Execute()
             }
             $this->CheckNavigationRights();
 
-            /** @var FlashBagInterface $flashBag */
-            $flashBag = $this->getCurrentRequest()->getSession()->getFlashBag();
-            if ($flashBag->has(MessageCheckOutput::CONSUMER_NAME)) {
-                $this->data['check_messages'] = $flashBag->get(MessageCheckOutput::CONSUMER_NAME);
-            }
-
```

Comment:
Warum fliegt das ersatzlos raus?

Reply:
Because we no longer show the sanity check output (or generate it). It made little sense to include these on each login.